x-pack/elastic-agent/docs/tab-widgets/run.asciidoc

```diff
@@ -92,7 +92,7 @@ cd 'C:\Program Files\Elastic-Agent'
 .\install-service-elastic-agent.ps1 <1> <2>
 ----
 <1> You must run {agent} under the SYSTEM account if you plan
-to use the {elastic-endpoint} integration.
+to use the {elastic-endpoint-integration} integration.
```

Comment:
this will be good to go once we update the `elastic-endpoint-integration` attribute.